### PR TITLE
Sync indicator with TextField

### PR DESCRIPTION
**Pull request by alejx**
- [ ] This change has been discussed with the author.
- [x] It fixes issue 492#.

---

**Description:**

Fix a critical problem that the indicator is not sync with text field in audio and subtitle quick settings panel.
This pr is not perfect, the position of the indicator is not correct. But this pr makes it easier to change the position of the indicator.

### CODE REVIEW

File: iina/QuickSettingViewController.swift [hunk 1]
```diff
@@ -369,19 +369,22 @@ class QuickSettingViewController: NSViewController, NSTableViewDataSource, NSTab
     }
   }
 
+  func redraw(_ indicator: NSTextField, constraint: NSLayoutConstraint, slider: NSSlider, value: Double) {
```
Comment (alejx) [inline iina/QuickSettingViewController.swift]: I'll move this function to a suitable place before merging.

**Comment by saagarjha** [comment]
Pull request #430 will probably cause merge conflict if it's merged in first, just saying.

**Comment by alejx** [comment]
@saagarjha I think you can migrate your changes into this pull request if need.

**Comment by saagarjha** [comment]
Please take a look at #430.

**Comment by alejx** [comment]
LGTM. Then I'll close this pr.

**Comment by saagarjha** [comment]
Mind if I delete the branch?

**Comment by alejx** [comment]
I'd like keep one day or so. And I'll clean it up.

**Pull request closed without merging**